### PR TITLE
update to spec 3010

### PR DESCRIPTION
**Pull request by carlosala**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -156,8 +188,8 @@ jobs:
           cd tests_zemu && yarn testSR25519
 
   build_package_0:
-    needs: [ configure, build, build_ledger, test_zemu, test_zemu_sr25519 ]
-    if: ${{ github.ref == 'refs/heads/master' }}
+    needs: [configure, build, build_nano_S, build_nano_SP, build_nano_X, test_zemu, test_zemu_sr25519]
+    if: ${{ github.ref == 'refs/heads/main' }}
```
Comment (ftheirs) [inline .github/workflows/main.yml:192]: default branch in Polymesh is master

File: README.md [hunk 2]
```diff
@@ -5,7 +5,8 @@
 
 ---
 
-![zondax](docs/zondax.jpg)
+![zondax_light](docs/zondax_light.png#gh-light-mode-only)
+![zondax_dark](docs/zondax_dark.png#gh-dark-mode-only)
```
Comment (ftheirs) [inline README.md:9]: light and dark mode images are missing

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -217,8 +249,8 @@ jobs:
         shell: bash -l {0}
         run: |
           source $HOME/.cargo/env
-          make SUBSTRATE_PARSER_FULL=1
-          mv ./app/pkg/installer_s.sh ./app/pkg/installer_nanos_xl.sh
+          make SUBSTRATE_PARSER_FULL=1 DISABLE_PREVIOUS=1
+          mv ./app/pkg/installer_s.sh ./app/pkg/installer_nanos_xl_current.sh
       - name: Set tag
         id: nanos_xl
         run: echo ::set-output name=tag_name::$(./app/pkg/installer_nanos_xl.sh version)
```
Comment (ftheirs) [inline .github/workflows/main.yml:256]: installer_nanos_xl_current.sh should be used

File: tests_zemu/tests/standard.test.ts [hunk 4]
```diff
@@ -358,7 +351,33 @@ describe('Standard', function () {
       const pathChange = 0x80000000
       const pathIndex = 0x80000000
 
-      const txBlob = Buffer.from(txJurisdiction, 'hex')
+      const txBlob = Buffer.from(txUtility_batch, 'hex')
```
Comment (ftheirs) [inline tests_zemu/tests/standard.test.ts:354]: txIdentity_addClaim should be used here

File: .github/workflows/main.yml (hunk 3 above)
Comment (ftheirs) [inline .github/workflows/main.yml:256]: same issue as above

Comment (carlosala) [reply .github/workflows/main.yml]: ups, done!

File: tests_zemu/tests/standard.test.ts (hunk 4 above)
Comment (carlosala) [reply tests_zemu/tests/standard.test.ts]: you're right, done!

**Review by ftheirs — APPROVED** [review]

**Pull request merged**